### PR TITLE
update launch with new runner, without rosbag signal

### PR DESCRIPTION
**Pull request by ipa-rwu**

### CODE REVIEW

File: mobika2_tests/config/test_5_with_nav.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  obstacle_tf2_broadcaster_node:
+    goal:
+      child_frame_id: obstacle
+      frame_id: map
+      position:
+        x: -1
+        y: 0
+        z: 0
+      yaw: 1.57 # degree
```
Comment (ipa-rwu) [inline mobika2_tests/config/test_5_with_nav.yaml]: @ipa-kut I will remove this part. We trust mavelmind will provide this frame

Comment (ipa-kut) [reply mobika2_tests/config/test_5_with_nav.yaml]: Do you want to remove the obstacle_tf2_broadcaster_node completely?

**Comment by ipa-rwu** [comment]
If we want to use this part, we need use code from https://github.com/ipa320/araig_test_stack/pull/53. Either I remove this part, to you rebase your branch on this branch

File: mobika2_tests/config/test_5_with_nav.yaml (hunk 1 above)
Comment (ipa-rwu) [reply mobika2_tests/config/test_5_with_nav.yaml]: If we want to use this part, we need use code from ipa320/araig_test_stack#53. Either I remove this part, to you rebase your branch on this branch

**Pull request merged**